package.json

```diff
@@ -23,8 +23,14 @@
   },
   "dependencies": {
     "@azure/core-asynciterator-polyfill": "^1.0.2",
+    "@babel/core": "^7.25.2",
     "@babel/plugin-proposal-decorators": "^7.20.7",
     "@babel/plugin-proposal-export-namespace-from": "^7.18.9",
+    "@babel/plugin-transform-class-properties": "^7.24.7",
```

Comment:
I don't think babel should be required in dependencies rather than devDependencies

Is there a reason these are being moved?

Reply:
Yes building was failing due to missing dependencies, I'm not sure why, and followed that comment
https://github.com/facebook/create-react-app/issues/13189#issuecomment-1686243701

Reply:
I agree this is not ideal but wonder if there is any risk?

Reply:
Bundle size would be the biggest concern, and if this workaround is masking a different issue